src/shell/lean.cpp

```diff
@@ -546,6 +536,8 @@ int main(int argc, char ** argv) {
         }
     }
 
+    lean::io_mark_end_initialization();
```

Comment:
Now `init_search_path` (L 436) runs under `IO,initializing = false`. This didn't seem to cause trouble for me.

Reply:
Please add this remark as a comment it the code.